applications/DEMApplication/custom_elements/spheric_particle.h

```diff
@@ -121,37 +121,37 @@ virtual std::unique_ptr<ParticleDataBuffer> CreateParticleDataBuffer(SphericPart
 
 void TransformNeighbourCoorsToClosestInPeriodicDomain(ParticleDataBuffer & data_buffer);
 void TransformNeighbourCoorsToClosestInPeriodicDomain(ParticleDataBuffer & data_buffer,
-                                                      const array_1d<double, 3>& coors,
-                                                      array_1d<double, 3>& neighbour_coors);
+                                                    const array_1d<double, 3>& coors,
+                                                    array_1d<double, 3>& neighbour_coors);
 void TransformNeighbourCoorsToClosestInPeriodicDomain(const ProcessInfo& r_process_info,
-                                                      const double coors[3],
-                                                      double neighbour_coors[3]);
+                                                    const double coors[3],
+                                                    double neighbour_coors[3]);
 
 virtual bool CalculateRelativePositionsOrSkipContact(ParticleDataBuffer & data_buffer);
 
 using DiscreteElement::Initialize; //To avoid Clang Warning. We tell the compiler that we are aware of the existence of this function, but we overload it still.
```

Comment:
I thin this can now be removed as we have the baseclass like this

Reply:
Do you mean the line
`using DiscreteElement::Initialize;`  ?

Reply:
yes

Reply:
I just removed it and that made me see that one file should not be there, so it was removed too. Thanks for your comments @philbucher .